torchtitan/models/llama/model.py

```diff
@@ -283,12 +290,55 @@ def __init__(self, layer_id: int, model_args: ModelArgs):
         self.n_heads = model_args.n_heads
         self.dim = model_args.dim
         self.attention = Attention(model_args)
-        self.feed_forward = FeedForward(
-            dim=model_args.dim,
-            hidden_dim=4 * model_args.dim,
-            multiple_of=model_args.multiple_of,
-            ffn_dim_multiplier=model_args.ffn_dim_multiplier,
-        )
+        self.enable_moe = model_args.enable_moe
```

Comment:
minor nit - I think this is expressed better as self.is_dense_model and self.is_moe_model. 'Enable' is not the same thing as 'is' so I think cleaner to express it's state of being via 'is'. 
In addition, I think the later code is cleaner expressed as self.is_dense_model makes for easier to read the checks vs 'not self.enable_moe', which may not later really mean it's a dense model if we start enabling other arches.